src/pg/pg-store.ts

```diff
@@ -491,20 +490,18 @@ export class PgStore extends BasePgStore {
   }
 
   async getSignerDataForBlock({ sql, blockId }: { sql: PgSqlClient; blockId: BlockIdParam }) {
-    let blockFilter: Fragment;
```

Comment:
Work around for a type error due to the api-toolkit using an older version of the `postgres` library.
TODO: update postgres library in toolkit repo